sky/serve/serve_state.py

```diff
@@ -337,6 +341,7 @@ def _get_service_from_row(row) -> Dict[str, Any]:
         'requested_resources': pickle.loads(requested_resources)
                                if requested_resources is not None else None,
         'requested_resources_str': requested_resources_str,
+        'tls_encrypted': bool(tls_encrypted),
```

Comment:
For existing service, this return value will be None, and `bool(tls_encrypted)` can fail?

Reply:
Actually, `bool(None)` will return `False`:

```bash
python
Python 3.9.18 (main, Sep 11 2023, 13:41:44) 
[GCC 11.2.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> bool(None)
False
```

But I agree that it is confusing. Changed it to 0 🫡